test/index.test.js

```diff
@@ -1628,15 +1628,18 @@ jobs:
 
         it('returns a pull request with the given prNum', () => {
             githubMock.pullRequests.get.yieldsAsync(null,
-                { number: 1, head: { sha } }
+                { html_url: "https://github.com/'repoOwner/repoName'/pull/1",
```

Comment:
can you fix indentation so it looks more like:
```
{
    html_url: "https://github.com/'repoOwner/repoName'/pull/1",
    number: 1,
    head: { sha }
}
```

Reply:
Why is `'repoOwner/repoName'` in quotes?

Reply:
for the first point, lint won't let me continue if I don't format it the way above...

Reply:
Weird :[